Fixes and updates to Shale and other themes

KamalJDavis opened the pull request:
- coolhole-style.js is removed so the navbar will function normally
- updated slate for better and consistent spacing to bring it in line with the other themes.
- navbar branding is aligned for all themes via coolhole.css
- chat input box respects theme colors again

KamalJDavis committed:
- coolhole-style.js is removed so the navbar will function normally
- updated slate for better and consistent spacing to bring it in line with the other themes.
- navbar branding is aligned for all themes via coolhole.css
- chat input box respects theme colors again

cck37 commented on www/css/themes/less/shale/shale.less:
```diff
@@ -589,10 +568,29 @@ input[type="checkbox"] {
+
+#motdrow #motdwrap {
+    background-color: #181e21;
+    padding: 24px;
```
Is this padding unique to shale? Feels a bit strange to have this not applied to all themes

cck37 commented on www/css/themes/less/shale/shale.less:
```diff
@@ -589,10 +568,29 @@ input[type="checkbox"] {
+}
+
+#motdrow #motdwrap {
+    background-color: #181e21;
```
While I'm here, is there a variable for this color already? Or perhaps we could just use `lighten` fn for background color.

streeeggs commented on www/css/themes/less/shale/shale.less:
```diff
@@ -589,10 +568,29 @@ input[type="checkbox"] {
+#motdrow #motdwrap {
+    background-color: #181e21;
+    padding: 24px;
+}
```
question: Is this padding unique to shale? Feels a bit strange to have this not applied to all themes

nit: While I'm here, is there a variable for this color already? Or perhaps we could just use `lighten()` with the background color as the arg

KamalJDavis replied:
I honestly just missed that padding, I'll throw that in coolhole.css. I'm experimenting with the color functions now, since the MOTD is a well just like polls it does just make sense to use lighten, darken, etc... to change their background colors.

KamalJDavis replied:
![Screenshot 2025-06-08 at 11 39 02 AM](https://github.com/user-attachments/assets/7ccafd51-fa0d-4a6d-b07f-066919d89aed)
Ok, so the ".well" class colors are handled in the variables file. Shale's polls have better color coordination in regard to their state.

streeeggs reviewed (commented):
Just looking at the code looks alright to me. would need to test it on my machine to confirm. The padding is my only real concern. Using variables for colors is a nice to have that we could update sometime in the future.

esnyder232 commented on www/css/themes/less/shale/shale.less:
```diff
@@ -573,9 +551,10 @@ input[type="checkbox"] {
 }
 
 .qe_title {
+  padding: 10px 0;
```
Not really a fan of the fat queue items.
I can kinda see why you added though, because it looks kind of claustraphobic without the spacing (plus the expanding animation does a little "jump").

It doesn't look bad, it would just be nice if you could get it skinnier.
I say we can still deploy and just see if people like/dislike it anyway.

KamalJDavis replied:
I was thinking they looked a little too big at times, the font size would affect this as well so I'll decrease that a few pixels to see if that helps balance it out.

streeeggs replied:
Pulled down today and noticed that it "feels" much bigger than other themes like modern. Here's a comparison showing ~20px difference in height between shale and modern:
![image](https://github.com/user-attachments/assets/cd0c6fe2-9545-4b12-b0ee-e61590e4f204)
![image](https://github.com/user-attachments/assets/235ec816-4dca-43bd-8343-99489692a814)

esnyder232 reviewed (commented):
Overall, everything looks fine to me.
I'd like to wait for streeeggs to also look at it on his localhost whenever he has time.

KamalJDavis committed:
- setting colors as less variables for easier usage in theme files
- the queue has a more appropriate font size, and less fat spacing/margains for queue items and respective button group
- changed the theme's font to Public Sans

streeeggs commented on www/css/coolhole.css:
```diff
@@ -1504,4 +1504,13 @@ code.text-lottery {
+}
+/* Theme wide fix for the navbar-brand */
+.navbar-brand {
+    padding: 5px 10px  !important;
```
fix (but I get it): Iknow it's probably obvious but `!important` at all costs. It'll just lead to a miserable chain of specificity.

I did notice that `@navbar-padding-horizontal` and `@navbar-padding-vertical` are variables, computed here:
https://github.com/streeeggs/coolhole/blob/697fb7ea9a5813cdf79a7a0fe72497b7fd6a89ee/www/css/themes/less/3.3.1/variables.less#L353-L354
and are used here:
https://github.com/streeeggs/coolhole/blob/697fb7ea9a5813cdf79a7a0fe72497b7fd6a89ee/www/css/themes/less/3.3.1/navbar.less#L166

Now, I don't envy the position of figuring out how to get `((@navbar-height - @line-height-computed) / 2)` and `floor((@grid-gutter-width / 2))` to spit out `5px 10px` for vertical and horizontal respectively. I'm not even sure if it's worth the hassle.
If anything, it might be worth looking into changing the logo size. I noticed when changing from a height of 40px to 20px, it looks ok:
![image](https://github.com/user-attachments/assets/4e5f7c93-da90-4d3b-89ab-2a5187860fb9)
Either that or perhaps we just hard-code padding we like in `navbar.less`. Your call; just avoid `!important` please lol

KamalJDavis replied:
The reason I did this is because the logo would be shifted down on every other theme, but it was fine on shale. I agree avoiding ```!important```, so I added more specificity to it so that way it will take precedence over the other themes stylings of ```navbar-brand``` without ```!important```.

streeeggs commented on www/css/themes/less/shale/shale.less:
```diff
@@ -589,10 +569,24 @@ input[type="checkbox"] {
+    background-image: linear-gradient(0deg, #0f2036, #1a1a1a);
+}
+
+/* Poll Themeing */
+
+
+// #pollwrap .well .wrapInner {
+//   background: #12151B;
+
+// }
```
nit: remove

streeeggs commented on www/css/themes/shale.css:
```diff
@@ -886,7 +886,7 @@ html {
+  font-family: "Public Sans", Helvetica, Arial, sans-serif;
   font-size: 14px;
   line-height: 1.42857143;
   color: #ffffff;
```
note: Was brought up that "shale's font color is too bright"
![image](https://github.com/user-attachments/assets/3800871a-742a-45c6-ad9a-9540eff660a6)

might be a good chance to change that here?

KamalJDavis replied:
Had to turn down the used for text color, it was pretty high by default
![Screenshot 2025-06-09 at 7 10 41 AM](https://github.com/user-attachments/assets/fecdb33b-7d36-49e5-b507-a1b82747fcb6)

streeeggs commented on www/css/themes/shale.css:
```diff
@@ -6430,7 +6531,7 @@ td > a:hover {
 }
 .navbar.navbar-inverse {
-  background-color: transparent;
+  background-color: #12151B;
```
todo: not to beat the dead hard-coded-colored-horse any more but I'd like to use this an example: using variables here could save our ass from needing to find/replace `#12151B` 13 times 
![image](https://github.com/user-attachments/assets/63de5a5f-2455-460c-9537-515f968159b4)
Or, worse, needing to adjust each color slightly to accommodate for major color changes. I think an ideal world would needing to adjust a color for texts, background, primary, secondary, and accent while everything else is inferred by those. 
Design systems like MUI saw this and built mechanisms to just do it for you: https://m2.material.io/inline-tools/color/
Just some food for thought though; not asking for an overhaul now and certainly not asking to move away from less and bootstrap

KamalJDavis replied:
lol, it is already a variable I must've missed it that one while making changes. It can be removed out right since that variable will auto apply
![Screenshot 2025-06-09 at 7 13 16 AM](https://github.com/user-attachments/assets/dedf5cd3-875e-49e9-b070-21218f9d1749)

streeeggs reviewed (commented):
![image](https://github.com/user-attachments/assets/179027ed-fd03-4964-a439-334cc97371b2)
Slate looks a little rough. Doubt anyone cares. otherwise most looks alright to me

KamalJDavis committed:
- text is less bright
- !important removed in favor of specificity
- removed uneeded comments

KamalJDavis committed:
- color removed from qe_title
- added mix-blend-mode to _disabled
